requirements.txt

```diff
@@ -1,26 +1,26 @@
-Django>=1.8,<1.9
+Django==1.10.5
 Whoosh==2.7.4
 django-auth-ldap==1.2.8
 django-form-utils==1.0.3
-django-haystack==2.4.1
-django-reversion==1.9.1
+django-haystack==2.6.0
+django-reversion==2.0.8
 django-tastypie==0.13.3
 psycopg2
 pyelasticsearch==1.4
```

Comment:
Oh, I thought we switched back to whoosh. Any idea when this will break? If we have some time we might as wel postpone until we use postgres full text search (#136).